hooks/useProposals.ts

```diff
@@ -104,7 +104,7 @@ const createFetcher =
     );
   };
 
-export const useL1Proposals = () => {
+export const useL1Proposals = ({ fetchProposalState }: { fetchProposalState?: boolean }) => {
```

Comment:
It seems we use this once and it's set to true, did I miss something? If we don't use, I'd prefer to cut the param.

Reply:
Yes, that is true. The idea is if we do reuse it we don't want the state fetch to always be bundled with the proposals. I can remove

Reply:
https://github.com/ScopeLift/l2-flexible-voting-frontend/pull/154/commits/2f2c309eb14c80b88ac5a6a31b3f57795612f6de